learn/configuration/instance_options.md

```diff
@@ -169,15 +171,52 @@ Sets the directory where Meilisearch will create dump files.
 
 ### Import dump
 
+::: warning
+This is a CLI flag and does not take any values. Assigning a value will throw an error.
+:::
+
 **Environment variable**: N/A
 **CLI option**: `--import-dump`
-**Default value**: `none`
-**Expected value**: a filepath pointing to a `.dump` file
+**Default**: Disabled
 
-Imports the dump file located at the specified path. Path must point to a `.dump` file.
+Replaces `data.ms` with dump data and starts the server using the provided dump. Path must point to a `.dump` file.
 
 Meilisearch will only launch once the dump data has been fully indexed. The time this takes depends on the size of the dump file.
 
+If the flag is specified when a database exists, an error occurs in the CLI.
```

Comment:
Unless I completely missed something, `--import-dump` is not a flag. It needs a path to a valid dump file, no?

Reply:
It says so in the [specification](https://github.com/meilisearch/specifications/pull/105/files). Section 1.3.6, line 114

Reply:
hmmm…I believe the definition of `flag` we are using in the docs (flags are options that do not accept any values) does not seem to be the same one used in the specs.

From the spec (emphasis mine):

> […] MeiliSearch will replace the data.ms with the dump data and start the server _using the provided dump_. e.g. `--import-dump dumps/20220117-144855452.dump`.

A dump must be provided and, as the example in that quote shows, it must be provided with `--import-dump`. Meaning, according to the definition we are using of `options` as instance options that accept a value and `flags` as instance option that do not accept a value, `--import-dump` is an option.

This does indicate we shouldn't rely on our chosen usage of those two words when speaking about value/value-less instance options. @dichotommy, what do you think?

Reply:
Yes, I agree w/ @guimachiavelli . In the wide world of software development "command-line option" and "command-line flag" are sometimes used interchangeably, the same way @gmourier used them in his spec. We're enforcing a distinction between the two: options take arguments/values, flags do not. According to this definition, `--import-dump` is not a flag: it takes a value.

[This Stack Exchange answer](https://unix.stackexchange.com/questions/285575/whats-the-difference-between-a-flag-an-option-and-an-argument#:~:text=An%20option%20can%20itself%20take,represent%20boolean%20on%2Doff%20switches.) is a nice reference IMO.